go/merkletree/merkle_verifier.go

```diff
@@ -0,0 +1,176 @@
+package merkletree
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+)
+
+// MerkleVerifier is a class which knows how to verify merkle inclusion and consistency proofs.
```

Comment:
The RFC refers to inclusion proofs as "audit proofs". Do we consistently use the term "inclusion proof" instead?